src/components/BorrowerProfile/LoanUse.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+	<span class="text-h2 text-gray-800">
+		{{ loanUseFiltered }}
+	</span>
+</template>
+
+<script>
+const loanUseFilter = require('../../plugins/loan-use-filter');
+
+export default {
+	props: {
+		loanAmount: {
+			type: String,
+			default: '',
+		},
+		use: {
+			type: String,
+			default: '',
+		},
+		status: {
+			type: String,
+			default: '',
+		},
+		name: {
+			type: String,
+			default: '',
+		},
+		borrowerCount: {
+			type: Number,
+			default: 1,
+		}
+	},
```

Comment:
Just wondering, is there a reason you decided to passing in multiple props, instead of just the loan object?